carbon-design-system/carbon
# fix(DatePicker): Fix minDate and maxDate reset

Undefined now is acceptable as value for minDate and maxDate after setted up a value

Closes #12270

After you had set up a value for the minDate and the maxDate it wasn't possible to reset them, because the relative useEffect accepted only non-falsy values.
Now the useEffect accept also undefined or null

#### Changelog

**Changed**

- useEffect for maxDate and minDate in DatePicker.js accept now falsy values

#### Testing / Reviewing
- Create a DatePicker component with no maxDate and minDate
- Click a button and set the minDate and maxDate
- Check the effect, by opening the date picker
- Click another button to set again minDate and  maxDate on undefined
- Check again the effect by opening the date picker
